person.rb

```diff
@@ -2,40 +2,41 @@
 # Create a class person
 
 class Person < Nameable
-    def correct_name
-        @name
-    end
-
-    def initialize(id, name = 'Unknow', age, parent_permission: true)
-        @id = Random.new.rand(1..100)
-        @name = name
-        @age = age
-        @parent_permission = parent_permission
-    end
-
-    # create getters and setters for @id, @name, and @age
-    attr_accessor :name, :age
-    attr_reader :id
-
-    # Create a private method named "is_of_age?" that returns true if age is greater or equal to 18 and false otherwise
-    private
-    def is_of_age?
-        @age >= 18
-    end
-
-    # Create public method named "can_use_services?" that returns true if person is of age or if they have permission from parents.
-    def can_use_services?
-        is_of_age? || @parent_permission
-    end
+  def correct_name
+    @name
+  end
+
+  super initialize(_id, name = 'Unknow', age, parent_permission: true)
+  @id = Random.new.rand(1..100)
+  @name = name
+  @age = age
+  @parent_permission = parent_permission
+end
+
+# create getters and setters for @id, @name, and @age
+attr_accessor :name, :age
+attr_reader :id
+
+  # Create a private method named "is_of_age?" that returns true if age is greater or equal to 18 and false otherwise
+  private
+
+def of_age?
+  @age >= 18
+end
+
+# Create public method named "can_use_services?"
+# that returns true if person is of age or if they have permission from parents.
+def can_use_services?
+  is_of_age? || @parent_permission
 end
 
-person = Person.new()
+person = Person.new
 person.is_of_age?
 person.can_use_services?
 
 person = Person.new(22, 'maximilianus')
-  person.correct_name
-  capitalizedPerson = CapitalizeDecorator.new(person)
-  capitalizedPerson.correct_name
-  capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
-  capitalizedTrimmedPerson.correct_name
\ No newline at end of file
+person.correct_name
+capitalized_person = CapitalizeDecorator.new(person)
+capitalized_person.correct_name
+capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)
+capitalized_trimmed_person.correct_name
```

Comment:
- [ ] I am not able to run these codes please address the issue and I suggest that you add in the README how to start the project

Reply:
Kindly, can you write more details, please!
I'm not sure I understand what you said

Thanks

Reply:
@DeliceLydia, what should I do with this pull request that already created?